frontend/pages/docs/users/managing-models-and-releases/upload-to-bailo/create-a-release.mdx

```diff
@@ -0,0 +1,24 @@
+import Image from 'next/legacy/image'
+import Box from '@mui/material/Box'
+import bailoNewRelease from 'public/docs/bailo_new_release.png'
+import bailoNewReleaseForm from 'public/docs/bailo_new_release_form.png'
+import DocsWrapper from 'src/docs/DocsWrapper'
+
+## **Creating a release.**
+
+Once you have uploaded a model, go to the **_Releases_** tab, and click on **_Draft new Release_**.
+
+<Box sx={{ maxWidth: '75%', margin: 'auto' }}>
+  <Image src={bailoNewRelease} />
+</Box>
+
+This will bring up a form that will require you to fill out a couple of fields and gives you the option of uploading
```

Comment:
"This will bring up a small form that will require you to choose a semver, add release notes and gives you the option of uploading files and images.